tests/runner/test_dataset_block.py

```diff
@@ -103,6 +127,113 @@ def test_partial_block_for_chunked_data(slicing_dim: Literal[0, 1, 2], last_in_c
     assert block.slicing_dim == slicing_dim
 
 
+@pytest.mark.parametrize("slicing_dim", [0, 1, 2])
+def test_partial_block_for_chunked_data_with_padding_center(
+    slicing_dim: Literal[0, 1, 2]
+):
+    block_shape = [10, 10, 10]
+    block_shape[slicing_dim] = 6
+    start_index = 3
+    data = np.ones(block_shape, dtype=np.float32)
+    global_index = [0, 0, 0]
+    global_index[slicing_dim] = 10 + start_index
+    global_shape_t = [10, 10, 10]
+    global_shape_t[slicing_dim] = 30
+    global_shape = make_3d_shape_from_shape(global_shape_t)
+    chunk_shape_t = [10, 10, 10]
+    chunk_shape_t[slicing_dim] += 4  # for padding
+    chunk_shape = make_3d_shape_from_shape(chunk_shape_t)
+    angles = np.linspace(0, math.pi, global_shape[0], dtype=np.float32)
+    block = DataSetBlock(
+        data=data,
+        aux_data=AuxiliaryData(angles=angles),
+        slicing_dim=slicing_dim,
+        block_start=start_index,
+        chunk_start=10,
+        global_shape=global_shape,
+        chunk_shape=chunk_shape,
+        padding=(2, 2),
+    )
+
+    assert block.is_padded is True
+    assert block.padding == (2, 2)
```

Comment:
Sorry for the nitpick: could you add a `padding = (2, 2)` var to be reused, just to make things a tad more explanatory, like in `test_partial_block_with_padding_global_boundaries`?